packages/appstore/src/components/modals/modal-manager.tsx

```diff
@@ -20,6 +20,7 @@ import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
 import FailedVerificationModal from './failed-veriification-modal';
 import AccountTransferModal from 'Components/account-transfer-modal';
 import RealWalletsUpgrade from './real-wallets-upgrade/real-wallets-upgrade';
+import CashierWalletModal from './cashier-wallet-modal';
```

Comment:
@george-usynin-deriv As I suggested above, since these modal and it's children components are related to cashier package , I suggest to move them all to the cashier page and import it here for further use.